### PR TITLE
Update wp org listing description

### PR DESCRIPTION
**Pull request by peterfabian**
### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This change should update the wp org description that shows up when you search for plugins in the context of wp.org plugin directory or via WP Admin Install Plugins screen:
 
Screenshot 2023-05-02 at 15 07 53

It can be accompanied by a similar update directly in the SVN repo to expedite the change.

Internal ref: p6riRB-8Aw-p2

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

I don't think there is any way to test this change locally, unless you want to create a new wporg plugin and test it out through the listing of the new plugin.

And since we're testing in production, this is how it looks live:

Screenshot 2023-05-03 at 22 50 42

### CODE REVIEW

File: plugins/woocommerce/readme.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ Stable tag: 7.6.1
 License: GPLv3
 License URI: https://www.gnu.org/licenses/gpl-3.0.html
 
-WooCommerce is the world’s most popular open-source eCommerce solution.
+WooCommerce has helped millions of merchants build successful businesses for the long term. Get everything you need to launch an online store in days and keep it growing for years. From your first sale to millions in revenue, Woo is with you.
```
Comment (barryhughes) [inline plugins/woocommerce/readme.txt]: Text is lovely, but wanted to note that at 242 chars (by comparison, the current description is 71 chars long) this exceeds the [recommended limited of 150 chars](https://developer.wordpress.org/plugins/wordpress-org/how-your-readme-txt-works/). The possible risk being, the text may be truncated in some situations.

As long as we're fine with that, this is good to merge (wanted to pause and check we're aware of the recommendation first, though).

Comment (peterfabian) [reply plugins/woocommerce/readme.txt]: Thanks, wasn't aware of it. Updated after consultation with @dlcallaway in [cd8dcf2](https://github.com/woocommerce/woocommerce/pull/38074/commits/cd8dcf29a711492f4029c1c977dd1a2c827a87bd)

**Review by barryhughes — APPROVED** [review]
LGTM!

**Comment by peterfabian** [comment]
Just noting that this has been requested for a code freeze exception for 7.7 and the change has been already done directly in SVN.

**Pull request merged**